.scala-steward.conf

```
@@ -0,0 +1,4 @@
+# Ignore updates to scala-xml, so it stays at 1.3.0 forever
+updates.ignore = [
+  { groupId = "org.scala-lang.modules", artifactId = "scala-xml" }
```

It supports version prefixes, so we can even keep it stewarding 1.4 if it ever materializes.  (I don't think it ever will, but the trick is handy.)

```suggestion
  { groupId = "org.scala-lang.modules", artifactId = "scala-xml", version = "1." }
```

---

Neat trick! I think for that to work it should be `updates.pin` instead of `updates.ignore` however.

---

Oh, yes, good catch!